core/src/utils/process.rs

```diff
@@ -67,6 +68,41 @@ pub fn check_for_process(name: &str) -> Result<()> {
                 }
             }
 
+            match listeners::get_processes_by_port(port) {
```

Comment:
Maybe I'm blind, but I can't see where `port` is actually used

Reply:
you mean where it comes from? it's a parameter to the function

Reply:
`listeners` is a crate which has `get_processes_by_port` as a static helper

Reply:
nvm I'm blind